src/util/middelwares.ts

```diff
@@ -15,6 +18,7 @@ import { UserInfo } from "../model/jwt";
     res.json("Hello " + req.UserInfo?.username);
   }
 );
+ * @deprecated
  */
 export function getUserInfo(req: express.Request, _res: express.Response, next: express.NextFunction) {
```

Comment:
isn't used anywhere in our code, can be deleted without breaking backwards compatibility

Reply:
Currently we use this function to test the transformer, its bad ik. For legacy reasons I would keep this deprecated Code if we need it for Express later on. If not we can delete it anyway.